a10_octavia/controller/worker/controller_worker.py

```diff
@@ -131,6 +131,7 @@ def create_health_monitor(self, health_monitor_id):
         :returns: None
         :raises NoResultFound: Unable to find the object
         """
+        time.sleep(60)
```

Comment:
Are we doing this because the database is not updated with the hm entries?
Please don't use sleep(60), we can not guarantee it will be done after 60 seconds and it is wasting time. Please:
- Write an general API which wait until the entry is ready in database. (by implement an retry mechanism and check every second)
- provide an option in configuration file for max timeout to wait the database entry.
- You can reference VThunderComputeConnectivityWait(), but we only need max timeout in this case.

Reply:
Removed sleep(60).
Added a "wait_for_db_entry()" api  in utils.py to check database entry every second.
provided an option "max_db_timeout" in configuration file for max timeout to wait the database entry.